up-l3/README.adoc

```diff
@@ -64,3 +64,35 @@ This special purpose built uEs are required to implement uProtocol and must be p
 |link:utwin/v2/README.adoc[*v2*]
 |Local cache of published events
 |===
+
+
+== Name & Number Registry
+
+In the following section we will elaborate on requirements for assigning names and numbers (ids) to uEs to avoid name space and number collisions.
```

Comment:
```suggestion
uProtocol URIs (UUri) contain a uEntity Identifier (`ue_id`) to refer to a particular service _type_. The `ue_id` is a 32 bit unsigned integer of which the least significant 16 bit are used to indicate the service type.

The following section defines sub-ranges of the 16 bit service type address space to be used for different purposes. This will make sure that custom service types defined in a private scope can co-exist with standard uProtocol service types without ambiguities and interference.
```